src/arr/compiler/compile-lib.arr

```diff
@@ -520,6 +520,13 @@ fun make-standalone(wl, compiled, options):
           end
       end
     end)
+
+  runtime-options = J.j-obj(
```

Comment:
@blerner here's a place that has an opportunity to compile in stuff about whether to print check results at all (referring to an earlier conversation about the four combinations of “check mode on/off” for both compile and runtime.  Compile in an options object into the program that handalone sees.